core/src/main/kotlin/plugability/DokkaContext.kt

```diff
@@ -22,6 +22,10 @@ interface DokkaContext {
     val logger: DokkaLogger
     val configuration: DokkaConfiguration
     val platforms: Map<PlatformData, EnvironmentAndFacade>
+    val pointsUsed: MutableSet<ExtensionPoint<*>>
+    val pointsPopulated: MutableSet<ExtensionPoint<*>>
```

Comment:
Here you are stating that points used and points populated are part of context interface, so every user of every context is able to add or remove used or populated points in any moment. Only read-only collection of unused points should be part of the api. The rest is just implementation detail.

Reply:
Removed.